pkg/image/helpers.go

```diff
@@ -0,0 +1,49 @@
+package image
```

Comment:
move to a subpackage.  even pkg/image/imageutil is swallowable.

Reply:
Done